CIP-1694/README.md

```diff
@@ -0,0 +1,664 @@
+
+* **DReps**<br/>
+  The DRep vote threshold that must be met as a percentage of *active voting stake*, ranging from 0 to 100 (inclusive).
+
+* **AVST**<br/>
+  The _**A**ctive **V**oting **S**take **T**hreshold_. The percentage to be used to determine if there is *sufficient active voting stake*.
+
+* **AVST fallback**<br/>
+  The fallback condition if the AVST threshold is not met
+  * **None**: there is no fallback, the action cannot be ratified unless the AVST threshold is met
+  * **SPO vote**: the action can be ratified if there is a sufficient SPO vote endorsement.
+
+
+* **SPOs**<br/>
+  The SPO vote threshold which must be met as a percentage of the stake held by all stake pools. The SPO vote is only considered if the AVST threshold is :x: or the AVST is below the AVST threshold.
+
+| Governance Action Type                               | Constitutional Committee | DReps    | AVST     | AVST Fallback | SPOs     |
+| ---                                                  | :---:                    | ---      | ---      | ---           | ---      |
+| 1. Motion of no-confidence                           | :x:                      | $P_1$    | $Q_1$    | None          | $R_1$    |
+| 2(a). New Committee/quorum (_normal state_)          | :heavy_check_mark:       | $P_{2a}$ | $Q_{2a}$ | SPO Vote      | $R_{2a}$ |
+| 2(b). New Committee/quorum (_state of no-confidence_)| :x:                      | $P_{2b}$ | $Q_{2b}$ | None          | $R_{2b}$ |
+| 3. Update to the Constitution                        | :heavy_check_mark:       | $P_3$    | $Q_3$    | SPO Vote      | $R_3$    |
+| 4. Hard-Fork initiation                              | :heavy_check_mark:       | $P_4$    | 0        | -             | $R_4$    |
+| 5. Protocol parameter changes                        | :heavy_check_mark:       | $P_5$    | $Q_5$    | SPO Vote      | $R_5$    |
+| 6(a). Treasury withdrawal, $[T_0, T_1)$              | :heavy_check_mark:       | $P_{6a}$ | $Q_{6a}$ | SPO Vote      | $R_{6a}$ |
+| 6(b). Treasury withdrawal, $[T_1, T_2)$              | :heavy_check_mark:       | $P_{6b}$ | $Q_{6b}$ | SPO Vote      | $R_{6b}$ |
+| 6(c). Treasury withdrawal, $[T_2, T_3)$              | :heavy_check_mark:       | $P_{6c}$ | $Q_{6c}$ | SPO Vote      | $R_{6c}$ |
+
+Some of the parameters given in this table ( $P_1$ ... $R_{6c}$, $T_1$ ... $T_3$ ) may be updatable protocol parameters, but others should be hard-coded. This proposal deliberately leaves both this choice and the choice of actual parameter values open for discussion.
+
+> **Note**
+> For all treasury withdrawals, the withdrawal threshold is the **total** amount of Lovelace that is withdrawn by the action, not the amount of any single withdrawal if the action specifies more than one withdrawal.
+
+##### Governance Action Lifecycle
+
+Governance actions are checked for ratification only on an epoch boundary. This delay allows everyone to vote on each proposal and prove that they are active.
+
```

Comment:
I feel like the governance action lifecycle is missing a notion of minimal persistence after the action has been ratified.
There will always be attempts to immediately repeal a ratified proposal. A minimal duration before a ratified action can be recanted might be necessary in order to avoid chaos.

Reply:
In this CIP, we are using the term "ratified" to mean that the action is now staged for enactment, and there is no notion of repeal. Are you maybe thinking of a situation where an action has met the thresholds, but is still waiting for the epoch boundary to be enacted? And there is a push to get people to change their votes?

Reply:
No, I was thinking more about putting a freeze on a given topic that was voted on previously. You don't want to be voting on the same thing every epoch.

Reply:
There are some natural delays with this, but it also seems possibly dangerous to forbid certain voting. Let's say something gets in and people realize it was a mistake, then having an extra delay on repealing it again is a bad thing. The CC should rather deal with this on a case-by-case basis.

Reply:
I don't think it is all that dangerous to limit having the same vote only every 5-6 epochs. Most ratified governance actions take time to be enacted anyway. 
Sure, the CC could deal with this. It really depends on how often this might be happening, which is very hard to estimate.

Reply:
> limit having the same vote only every 5-6 epochs

That's an entire month. There are definitely cases where every five days makes sense: treasury withdrawals and protocol parameter changes (imagine changing the fees parameters continually). And I see no problem with letting folks vote again right away on a contentious issue.

> Most ratified governance actions take time to be enacted anyway.

It will only take our system five days (one epoch).

Reply:
I misunderstood the process of enactment. It totally makes sense to allow new votes between the ratification and the enactment, if this time span is limited to one epoch. 

My initial comment should have been the following:
Should there be a minimal duration before another vote can be held about something that has been enacted already.

Reply:
> new votes between the ratification and the enactment

We might be still miscommunicating. Once an action is ratified (on an epoch boundary), no more votes for that action are accepted, and the action will be enacted at the following epoch boundary.

> Should there be a minimal duration before another vote can be held about something that has been enacted already.

Since only one action of each type can be ratified per epoch, there is now a built in five day period between changes. but we never stop voting on actions, by design.

Reply:
Sorry, for my ambiguous choice of words. I meant that it would make sense to have another vote via a new governance action that would allow a previously ratified action to be dropped before it can be enacted.

I thought that as per https://github.com/cardano-foundation/CIPs/pull/380#discussion_r1113269204 that the "one action of each type per epoch" limitation would be dropped.

Reply:
> have another vote via a new governance action that would allow a previously ratified action to be dropped before it can be enacted

I'm not sure that that would be worth the extra complication. We would need a compelling reason to offset the complexity.

> the "one action of each type per epoch" limitation would be dropped.

We have ideas for something to basically accomplish the same thing, more on that soon :)

Reply:
Was there an update on this thread?

Reply:
My previous comment in this thread was alluding to https://github.com/JaredCorduan/CIPs/blob/voltaire-v1/CIP-1694/README.md#restrictions.